config/sections/mixins/layout.php

```diff
@@ -145,30 +145,25 @@
 			// TODO: remove once Form classes have been refactored
 			// @codeCoverageIgnoreStart
 			if ($this->rawvalues === true) {
-				foreach ($this->columns as $columnName => $column) {
-					$item[$columnName] = match (empty($column['value'])) {
-						// if column value defined, resolve the query
-						false   => $model->toString($column['value']),
-						// otherwise use the form value,
-						// but don't overwrite columns
-						default => $item[$columnName] ?? $model->content()->get($column['id'] ?? $columnName)->value()
-					};
-				}
-
-				return $item;
+				$row = [
+					...$model->content()->toArray(),
+					...$item,
+				];
+			} else {
+				$row = [
+					...Form::for($model)->toFormValues(),
+					...$item,
+				];
```

Comment:
`Question:` Reading the code, I would think we have a breaking change here where the assignment from column id to content field name no longer works (before it was either the key of the column array or the `id` option inside the array). Do you know if this has any implications?